src/main/common/filter.c

```diff
@@ -101,13 +123,21 @@ FAST_CODE float pt2FilterApply(pt2Filter_t *filter, float input)
 
 FAST_CODE float pt3FilterGain(float f_cut, float dT)
 {
-    // PTn cutoff correction = 1 / sqrt(2^(1/n) - 1)
-    #define CUTOFF_CORRECTION_PT3 1.961459177f
-
     // shift f_cut to satisfy -3dB cutoff condition
     return pt1FilterGain(f_cut * CUTOFF_CORRECTION_PT3, dT);
 }
 
+// Calculates filter gain based on delay (time constant of filter) - time it takes for filter response to reach 63.2% of a step input.
+float pt3FilterGainFromDelay(float delay, float dT)
```

Comment:
This function needs a declaration in the header file

Reply:
done